cmd/traffic_cache_tool/CacheScan.cc

```diff
@@ -272,9 +278,14 @@ CacheScan::unmarshal(char *buf, int len, RefCountObj *block_ref)
   ink_assert(alt->m_writeable == 0);
   len -= HTTP_ALT_MARSHAL_SIZE;
 
+  // usually the fragment count is less or equal to 4
```

Comment:
Do you mean "normally less than or equal to HTTPCacheAlt::N_INTEGRAL_FRAG_OFFSETS"?